Assets/Scenes/editor.unity

```diff
@@ -4905,6 +4906,109 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 2da0c512f12947e489f739169773d7ca, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
+--- !u!1001 &758602354
```

Comment:
Kannst du bitte noch ein HoverLable(das ist eine Komponente) hinzufügen, das die Eingabe erklärt? Also 0: keine Backups, <0: unendlich Backups, >0: entsprechende Anzahl an Backups

Reply:
Please add a HoverLable(MonoBehaviour), to explain the entry.
0 -> No Backups
>0 -> Keep specific amount of Backups
-1 -> Keep infinite amount of Backups